config.json

```diff
@@ -172,6 +172,22 @@
       ]
     },
     {
+      "uuid": "a9c2fbda-a1e4-42dd-842f-4de5bb361b91",
+      "slug": "dot-dsl",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 5,
+      "topics": [
+        "equality",
+        "classes",
+        "lists",
+        "maps",
+        "object_oriented_programming",
+        "test_driven_development",
+        "transforming"
```

Comment:
What's the reasoning for including `maps`, `test_driven_development` and `transforming`?

Reply:
When I left `maps` there, I was thinking of dictionaries. Now that I think of it again, I think `maps` probably means something else, in this case.

For `object_oriented_programming`, I thought that since we were using classes, it was fair to leave that as a topic.

For `test_driven_development`, I think there isn't any other way to complete this other than by reading through the test cases and building up the DSL, so I thought TDD is accurate here.

As for `transforming`, I thought it was fair to say that DSLs are a transformation of one representation of data to another.

Reply:
Thanks for your explanation. I don't think that `maps` is a suitable topic here, but `domain_specific_languages` and `graphs` would probably be a sensible additions.

Reply:
Done!